core/src/epicli/data/common/ansible/playbooks/roles/haproxy_runc/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Assert input parameters
+  assert:
+    that:
+      - haproxy_service is defined
+
+- name: Install required system packages
+  include_tasks: "install-packages-{{ ansible_os_family }}.yml"
```

Comment:
Is it the correct place to install packages? Should it be in 'repository' role?

Reply:
Same as before.